core/src/main/java/io/paradoxical/cassieq/environment/Env.java

```diff
@@ -0,0 +1,14 @@
+package io.paradoxical.cassieq.environment;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME )
+public @interface Env {
+    String help();
+
+    String defaultValue();
```

Comment:
consider defaulting this to null or "" this way callers won't have to specify it